.github/deploy/db_token.ps1

```diff
@@ -0,0 +1,15 @@
+. $PSScriptRoot/steps/00-Config.ps1
+
+$tenantId = (az account show --query tenantId --out tsv)
+$authAppId = "069f8930-011e-4361-b065-ab63b120b351"
+$workspaceUrl = az resource show `
+  --resource-group $resourceGroupName `
+  --name $databricksName `
+  --resource-type "Microsoft.Databricks/workspaces" `
+  --query properties.workspaceUrl `
+  --out tsv
+
+atc_az_databricks_token `
+  --tenantId $tenantId `
+  --appId $authAppId`
+  --workspaceUrl $workspaceUrl
```

Comment:
Does this have something to do with the cache loader? 
You could consider splitting up the PR into multiple PRs which each implements a new feature. This both lets us track the changes, and makes it easier to find out what new features might cause challenges.

Reply:
fair point. i took it out